capsules/classifier_face_age_gender_openvino/config.py

```diff
@@ -0,0 +1,3 @@
+genders = ['feminine', 'masculine']
+
+#TODO: Potentially add age binning to the config for use in attributes
```

Comment:
Remove or resolve TODO

Reply:
I'd like this implemented before merging. It would enable people to create alarms on BrainFrame based on age (which isn't possible with data in extra_data)